lib/msal-browser/FAQ.md

```diff
@@ -128,6 +129,10 @@ Yes, MSAL.js 2.x can be used as a custom authentication provider for the [Micros
 
 Yes, we recommend the new [Powershell Graph SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell) for doing so. For instance, the script below creates an Azure AD application with redirect URI of type **SPA** and **User.Read** permission for Microsoft Graph in a tenant specified by the user, and then provisions a service principal in the same tenant based on this application object:
 
+## Can I acquire tokens using popups on mobile browsers
```

Comment:
Can we make this more general? i.e. "Does MSAL.js support mobile browsers?" and add a statement to the effect of "in general, yes ,however, given we do not test specifically test against mobile browsers support may be limited and some features and flows may not work. For problems specific to mobile browsers not listed here please open an issue with clear instructions to reproduce"

Reply:
And then move this to be right below the browser support question

Reply:
+1

Reply:
Updated, thanks for the feedback!